components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -3819,6 +3819,14 @@ components:
           type: boolean
           readOnly: true
           example: false
+        createdTime:
+          type: string
+          readOnly: true
+          example: 2021-02-11-09:57:25
```

Comment:
are we getting the time using this format or using the long timestamp format eg: 1651124030399?

Reply:
Getting timestamp as follows:
`{"count":1,"list":[{"applicationId":"2df86a22-e1d1-4317-bbcb-ead444cf5d20","name":"AppARR26","throttlingPolicy":"Unlimited","description":"aaaa","status":"APPROVED","groups":[],"subscriptionCount":0,"attributes":{},"owner":"admin","createdTime":"2022-04-27 17:55:27.363","updatedTime":"2022-04-27 18:02:02.112"}],"pagination":{"offset":0,"limit":25,"total":1,"next":"","previous":""}}`

Reply:
I think we'll have to change this to the Long timestamp format.. This will be confusing when showing to the user. Because we'll need to know some sort of Timezone and that is not available in the date. Since we are using the Long timestamp format in other places as well in the REST API, we can use that here as well.

Reply:
In the application creation path, we are setting this time in the `Timestamp` format: https://github.com/wso2/carbon-apimgt/blob/9.2.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L3604-L3606


ex: 2022-05-02 15:58:35.236

In other places also we use this format: https://github.com/wso2/carbon-apimgt/blob/9.2.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L5557, https://github.com/wso2/carbon-apimgt/blob/9.2.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L2331, https://github.com/wso2/carbon-apimgt/blob/9.2.x/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java#L5153

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/11241/commits/544baf0ef577f1b980377b711c9a0cedbc604055